src/main/java/org/jabref/logic/ai/summarization/SummariesService.java

```diff
@@ -56,34 +59,53 @@ public SummariesService(AiPreferences aiPreferences,
         this.taskExecutor = taskExecutor;
     }
 
-    /**
-     * Start generating summary of a {@link BibEntry}, if it was already generated.
-     * This method returns a {@link ProcessingInfo} that can be used for tracking state of the summarization.
```

Comment:
The `ProcessingInfo` is still retruned? Why this comment is deleted?

Reply:
Did some crazy refactoring. It got lost.

Better to return it